framework/test/unit/modules/reward/api.spec.ts

```diff
@@ -0,0 +1,112 @@
+import { getRandomBytes, hash } from '@liskhq/lisk-cryptography';
+import { RewardModule } from '../../../../src/modules/reward';
+import { EventQueue } from '../../../../src/node/state_machine';
+
+describe('RewardModuleAPI', () => {
+	const genesisConfig: any = {};
+	const moduleConfig: any = {
+		distance: 3000000,
+		offset: 2160,
+		brackets: [
+			BigInt('500000000'), // Initial Reward
+			BigInt('400000000'), // Milestone 1
+			BigInt('300000000'), // Milestone 2
+			BigInt('200000000'), // Milestone 3
+			BigInt('100000000'), // Milestone 4
+		],
+		tokenIDReward: { chainID: 0, localID: 0 },
+	};
+	const generatorConfig: any = {};
+
+	const context = {
+		getStore: jest.fn(),
+		eventQueue: new EventQueue(),
+	};
+
+	const blockAsset = {
+		getAsset: jest.fn(),
+	};
```

Comment:
it would be better if the jest mock is initalized in each `beforeEach` block, so that the counter is always reset (thus, no false testing)

Reply:
You are right, I moved `context` and `blockHeader` to `beforeEach`.